tests/unit/dataactbroker/test_file_routes.py

```diff
@@ -19,24 +19,27 @@ def file_app(test_app):
     yield test_app
 
 
-def test_is_period(file_app, database):
-    """Test listing user's submissions. The expected values here correspond to
-    the number of submissions within the agency of the user that is logged in
+def test_list_banners(file_app, database):
     """
-
-    application = ApplicationTypeFactory(application_name='fabs')
-    database.session.add(application)
+        Test listing user's submissions. The expected values here correspond to the number of submissions within the
+        agency of the user that is logged in
```

Comment:
This comment is inaccurate. We aren't checking user's submissions here and we don't expect the number of returns to have anything to do with the user or their submissions.

Also a style note, stylistically we keep the first line of the multi-line comment on the same line as the initial `"""`